### PR TITLE
Update Razor version

### PR DESCRIPTION
**Pull request by ajaybhargavb**
cc @NTaylorMullen @ryanbrandenburg @JoeRobich

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
   * As an alternative, consider installing the [MSBuild Project Tools](https://marketplace.visualstudio.com/items?itemName=tintoy.msbuild-project-tools) extension by @tintoy.
 
 ## 1.21.12 (Not Yet Released)
+* Updated Razor support
```
[the remark this answers is not in the document]
Comment (ajaybhargavb) [reply CHANGELOG.md]: Done

File: package.json [hunk 2]
```diff
@@ -498,17 +498,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```
Comment (ryanbrandenburg) [inline package.json]: As I recall they had these in there for a reason, you should probably add them back.

File: package.json [hunk 3]
```diff
@@ -498,17 +498,17 @@
...
+                  "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
                   "default": []
                 },
                 "searchMicrosoftSymbolServer": {
                   "type": "boolean",
-                  "description": "If 'true' the Microsoft Symbol server (https\u200b://msdl.microsoft.com\u200b/download/symbols) is added to the symbols search path. If unspecified, this option defaults to 'false'.",
```
Comment (ryanbrandenburg) [inline package.json]: Same as above for all `\u200b`.

File: package.json (hunk 2 above)
Comment (ajaybhargavb) [reply package.json]: Oh interesting, I'm curious to know what the reasoning behind this is. @JoeRobich why were these needed?

Comment (JoeRobich) [reply package.json]: > why were these needed?

I have no idea if they are needed. Our update package dependencies launch task will restore them prior to the next release. So no reason to do it here.

**Pull request merged**